core/src/test/java/org/pentaho/di/core/row/value/ValueMetaBaseTest.java

```diff
@@ -1111,4 +1117,368 @@ public void testHashCodes() throws Exception {
     valueMetaString.type = ValueMetaInterface.TYPE_NONE;
     assertEquals( valueMetaString.hashCode( "any" ),  0 );
   }
+
+  @Test
+  public void testGetMetadataPreview() throws SQLException, KettleDatabaseException {
+    ValueMetaBase valueMetaBase = new ValueMetaBase( );
+
+    DatabaseMeta dbMeta = spy( new DatabaseMeta() );
+
+    // SQL CHAR is a Pentaho String
+    ResultSet resultSet = mock( ResultSet.class );
```

Comment:
Use a private variable for `resultSet` and initialize it for every test in a `@Before` method.
This results in one less line for every test.